python/thunder/rdds/data.py

```diff
@@ -459,7 +459,7 @@ def count(self):
 
     def mean(self, dtype='float64', casting='safe'):
         """
-        Mean of values, ignoring keys
+        Mean of values across keys, returned as an ndarray.
```

Comment:
These are better but might be even more clear as "Mean of values computed by aggregating across records, returned as an ndarray". A little funny to say mean across keys, almost implies that we're adding up the keys. You could further add that the result will be the same size as a single record. Seem good? Change the others accordingly?